native/src/modules/categories/components/CategoryListContent.js

```diff
@@ -57,124 +53,41 @@ const CategoryListContent = ({
-            marginRight: config.hasRTLScript(language) ? listIndent : textDistanceToBullet,
-            marginLeft: config.hasRTLScript(language) ? textDistanceToBullet : listIndent
-          }}>
-          {index})
-        </Text>
-      )
-    },
-    [language]
-  )
-
-  // see https://github.com/archriss/react-native-render-html/issues/286
-  // TODO: remove with IGAPP-378
-  const renderLists = useCallback(
-    (htmlAttribs, children, convertedCSSStyles, passProps) => {
-      const { transientChildren, nodeIndex, key, listsPrefixesRenderers } = passProps
-      children =
-        children &&
-        children.map((child, index) => {
-          const rawChild = transientChildren[index]
-          let prefix = false
-          if (rawChild && rawChild.tagName === 'li') {
-            if (rawChild.parentTag === 'ul') {
-              prefix = listsPrefixesRenderers.ul(htmlAttribs, children, convertedCSSStyles, { ...passProps })
-            } else if (rawChild.parentTag === 'ol') {
-              prefix = listsPrefixesRenderers.ol(htmlAttribs, children, convertedCSSStyles, { ...passProps, index })
-            }
-          }
-          return config.hasRTLScript(language) ? (
-            <View key={`list-${nodeIndex}-${index}-${key}`} style={{ flexDirection: 'row' }}>
-              <View style={{ flex: 1 }}>{child}</View>
-              {prefix}
-            </View>
-          ) : (
-            <View key={`list-${nodeIndex}-${index}-${key}`} style={{ flexDirection: 'row' }}>
-              {prefix}
-              <View style={{ flex: 1 }}>{child}</View>
-            </View>
-          )
-        })
-      return <View key={key}>{children}</View>
-    },
-    [language]
-  )
+    }, [cacheDictionary])
 
   return (
     <SpaceBetween>
       <Container>
-        <Html
+        <RenderHTML
           source={{ html: content }}
-          onLinkPress={onLinkPress}
           contentWidth={width}
           defaultTextProps={{ selectable: true, allowFontStyling: true }}
           alterNode={alterResources}
-          listsPrefixesRenderers={{ ul: renderUnorderedListPrefix, ol: renderOrderedListPrefix }}
-          renderers={{ ul: renderLists, ol: renderLists }}
+          renderersProps={{
+            a: { onPress: onLinkPress },
+            ol: { enableExperimentalRtl: true },
+            ul: { enableExperimentalRtl: true }
+          }}
           baseFontStyle={{
             fontSize: 14,
             fontFamily: theme.fonts.native.contentFontRegular,
             color: theme.colors.textColor
           }}
           tagsStyles={{
-            p: { textAlign: config.hasRTLScript(language) ? 'right' : 'left' },
-            img: { align: config.hasRTLScript(language) ? 'right' : 'left' }
+            ul: { direction: config.hasRTLScript(language) ? 'rtl' : 'ltr' },
+            ol: { direction: config.hasRTLScript(language) ? 'rtl' : 'ltr' },
+            p: { textAlign: contentAlignment(language) }
```

Comment:
The `align` property for images is not needed anymore?

Reply:
The align property for images never actually existed or is not supported anymore. I think you can still set the alignment with proper html container and the text align property, but that's nothing we should need to worry about.